tests/integration/test_parsers.py

```diff
@@ -1,4 +1,4 @@
-from dug.parsers import DbGaPParser, TOPMedTagParser
+from src.dug.parsers import DbGaPParser, TOPMedTagParser
```

Comment:
this ended up being the fix for me when running `pytest .`... not sure how i feel about it.. I mean, it makes sense from a pathing standpoint.. but still. How was this working before?

Reply:
You shouldn't have to include src in the import path, that suggests your environment doesn't have dug in the path